openrtb/settlement.go

```diff
@@ -0,0 +1,10 @@
+package openrtb
+
+// Settlement is the interface that provides auction settlement details.
+//
+// Exchanges should implement this interface on the relevant auction detail objects.
+//
+// Price returns the auction settlement price.
+type Settlement interface {
```

Comment:
Yeah, I agree. It's just that the `MacroSub` interface call becomes a bit un-intuitive. When I call,

```go
openrtb.MacroSub(stringToSub, bidResp, settlement)
```

I think my expectation is to substitute the values from settlement to `stringToSub`, however, it also uses `bidResp`.